examples/config.yaml

```diff
@@ -5,6 +5,9 @@ config:
 
 spokes:
   - spoke1-name:
+      contrib:
```

Comment:
Are we sure that this will not affect the count we do in some parts of the code with the var ${i} or ${index}?

please check this, this is a dangerous change

Reply:
the query perform for each node in the spoke cluster is as follow:
```
yq e ".spokes[${spokenumber}].[]|keys" ${SPOKES_FILE} | grep master | wc -l | xargs
```

there are a grep filter with master key word

Reply:
![image](https://user-images.githubusercontent.com/9356454/167620732-78bbcbf8-7eec-4446-97ba-1c04a36ab77d.png)
We're always looking for master or worker